tests/bud.bats

```diff
@@ -6527,3 +6527,41 @@ _EOF
   expect_output --substring "localhost/foo/bar"
   expect_output --substring "localhost/bar"
 }
+
+@test "build test default ulimits" {
```

Comment:
is this test being added intentional? I see no code changes for this and it fails in CI so I assume no

Reply:
The rebase said it was necessary and I grabbed it in.  Let me go  yank it.